### PR TITLE
Feat: enable handler for the cancel streaming payment motion

### PR DESCRIPTION
**Pull request by rumzledz**
These changes are required to test [colonyCDapp#2519: Feat: enable Payment Stream cancellation via a Motion](https://github.com/JoinColony/colonyCDapp/pull/2519)

### CODE REVIEW

File: src/handlers/motions/motionCreated/handlers/streamingPayments.ts/cancelStreamingPaymentsMotion.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const [, , streamingPaymentId] = actionArgs;
+  const [, , domainId] = args;
+
+  if (!colonyAddress) {
+    return;
+  }
+
+  await createMotionInDB(event, {
+    type: motionNameMapping[name],
+    fromDomainId: colonyAddress
```
Comment (jakubcolony) [inline src/handlers/motions/motionCreated/handlers/streamingPayments.ts/cancelStreamingPaymentsMotion.ts:26]: nit-pick: `colonyAddress` is guaranteed to be truthy at this point

**Review by jakubcolony — APPROVED** [review]
LGTM 💪

File: src/handlers/motions/motionCreated/handlers/streamingPayments.ts/cancelStreamingPaymentsMotion.ts (hunk 1 above)
Comment (rumzledz) [reply src/handlers/motions/motionCreated/handlers/streamingPayments.ts/cancelStreamingPaymentsMotion.ts:26]: I shall clean that up! 👌

**Review by iamsamgibbs — APPROVED** [review]
Reviewed with CDapp#2519

**Review by rdig — APPROVED** [review]
Nothing sticks out codewise.

As for functional testing it was done as part of JoinColony/colonyCDapp#2519

**Pull request merged**